partner_report_open_invoices/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+============================
+Partner Report Open Invoices
+============================
+
+This module will create a report that contains the details of open invoices for partner in partner form view.
+
+
+Installation
+============
+
+This module depends on:
+
+* account
+
+Configuration
+=============
+
+* There needs a configuration of a message which will appear under Company Settings, just like Overdue Message.
+
+Usage
+=====
+
+* The module generates a report on partner which contains a list of outstanding journal entries and at the end it will
+show up the invoice messages configured inside the Company Configuration.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/92/8.0
+
+For further information, please visit:
```

Comment:
@JayVora-SerpentCS, yes, I mean that part because it's no longer present in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst